pkg/storegateway/bucket.go

```diff
@@ -893,6 +897,15 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 
 	blocks := s.blockSet.getFor(req.MinTime, req.MaxTime, req.MaxResolutionWindow, reqBlockMatchers)
 
+	spanLogger := spanlogger.FromContext(srv.Context(), s.logger)
+	level.Debug(spanLogger).Log(
+		"msg", "BucketStore.Series",
+		"min time", time.UnixMilli(req.MinTime).UTC().Format(time.RFC3339Nano),
+		"max time", time.UnixMilli(req.MaxTime).UTC().Format(time.RFC3339Nano),
+		"matchers", storepb.PromMatchersToString(matchers...),
+		"shard selector", maybeNilShard(shardSelector).LabelValue(),
```

Comment:
Since the store-gateway deals a lot with blocks, I would clarify what's timing here.


```suggestion
		"request min time", time.UnixMilli(req.MinTime).UTC().Format(time.RFC3339Nano),
		"request max time", time.UnixMilli(req.MaxTime).UTC().Format(time.RFC3339Nano),
		"request matchers", storepb.PromMatchersToString(matchers...),
		"request shard selector", maybeNilShard(shardSelector).LabelValue(),
```